src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,447 @@
+    these scans may be grouped into the same session.
+    In other datasets, a subject leaving the scanner and returning under the same
+    injection may be considered separate sessions.
+
+1.  **Run** - In PET, subjects may have to leave the scanner to use the bathroom.
+    While leaving the scanner would interrupt an MR acquisition, in PET this
+    disruption is more appropriately considered missing data during a run.
+
+1.  **Time zero** - A reference point in time, to which all timestamps pertaining
+    to a recording are relative.
+    Time zero will most commonly be the time of injection of a radioisotope, or
+    the time at which the first scan of an acquisition is begun.
+    If a pharmacological within-scan challenge is performed,
+    another time zero may be more convenient.
+
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined "time zero".
+
+![Figure 1](images/PET_scan_overview.svg "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, "time zero" can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+In this example, tracer injection coincides with scan start.
+
+## PET recording data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    pet/
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.nii[.gz]
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.json
+```
+
+PET data MUST be stored in the `pet` directory.
+PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired)
+NIfTI files with the `_pet` suffix.
+Volumes MUST be stored in chronological order (the order they were acquired in).
+
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) is used to
+indicate a task subjects were asked to perform in the scanner.
+Those labels MUST be consistent across subjects and sessions.
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity is used to
+indicate the tracer used.
+This entity is OPTIONAL if only one tracer is used in the study,
+but REQUIRED to distinguish between tracers if multiple are used.
+The label used is arbitrary and each file requires a separate JSON sidecar
+with details of the tracer used (see below).
+Examples are `acq-18FFDG` for fludeoxyglucose or `acq-11CPIB` for Pittsburgh compound B.
+Other labels are permitted, as long as they are consistent across subjects and sessions
+and consist only of the legal label characters.
+
+If more than one run of the same task and acquisition (tracer) are acquired during
+the same session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
```

Comment:
Is 0 padding recommended here to label the runs? in other modalities there could be tens/hundreds of runs, not sure here

```suggestion
`_run-01`, `_run-02`, `_run-03`, and so on.
```

Reply:
The current non-padded version is consistent with the rest of the spec.

Reply:
So the common thing is to have very few runs (<9)?

Reply:
Yes. Usually just a single run, but few cases have more runs. I have never seen more than 3.

Reply:
I hadn't thought of it that way, but yes, I don't know that I've seen many datasets with 10 or more runs of the same task in a session. Zero-paddidng is permitted, but is not specifically recommended.

My point was just that if we want to adjust the wording to use zero-padding, we should do it consistently across the spec, not just in PET because that's the current PR.

Reply:
Ah! If there are just this few it makes sense :)

Reply:
@effigies you are right, this recommendation is mainly for sessions, e.g. longitudinal studies: 
https://bids-specification.readthedocs.io/en/stable/06-longitudinal-and-multi-site-studies.html

Reply:
Ok, we will not zero pad then.